packages/plugins/users-permissions/server/controllers/content-manager-user.js

```diff
@@ -147,7 +147,7 @@ module.exports = {
         .query('plugin::users-permissions.user')
         .findOne({ where: { username } });
 
-      if (userWithSameUsername && userWithSameUsername.id != id) {
+      if (userWithSameUsername && _.toString(userWithSameUsername.id) !== _.toString(id)) {
```

Comment:
I think we already discussed it but I don't remember. Why not `String(...)` instead of `_.toString(...)`?

Reply:
A few things differentiate them (Not really a better one mostly a question of preference/consistentcy in the codebase)

- String will convert undefined and null to 'undefined' and 'null', -0 will become 0
- toString will convert undefined and null to '', -0 will become -0

However the one thing that for me makes a bigger diff is how explicit the code is. When using `String(..)` since String is a constructor you could say. why aren't we using `new String()` which seems the right thing to do when using Constructor. 

But in that case new String will not return a string primitive but an object which isn't what you would want. The Constructor can have 2 different behaviors when toString is always behaving the same way and as only one way to be invoked. 

This can be avoided with linters though and we do use Boolean() in the same manner so we can't say we are perfectly consistent to be fair.